app/models/event.rb

```diff
@@ -15,6 +15,8 @@ class Event < ApplicationRecord
 
   before_save :set_updated_event_at, if: :public_fields_updated?
 
+  after_update :send_notification
```

Comment:
There's already another PR for this, please make only one Story per PR

Reply:
OK, I remove this feature